lib/sidekiq_autoscale/sidekiq_interface.rb

```diff
@@ -9,7 +9,12 @@ def total_queue_size
     end
 
     def queue_names
-      ::Sidekiq::Queue.all.map(&:name)
+      from_env = ENV["SIDEKIQ_QUEUES"]
+      if from_env
```

Comment:
```suggestion
      if (from_env = ENV["SIDEKIQ_QUEUES"])
```

In ruby, this (`if (var = maybe_fasley)`) is called the "safe-assignment pattern." It's used when a variable is only used inside of a `if` block if and only if it isn't `nil` or `false`. As in this case.